styles.css

```diff
@@ -1,122 +1,157 @@
-    /* border: 0.5em solid black; */
-    display: flex;
-    flex-direction: row;
-    justify-content:space-between;
-    height: 95vh;
-    color: #FFF;
-    background-color: #46424D;
+  /* border: 0.5em solid black; */
+  display: flex;
+  flex-direction: row;
+  justify-content:space-between;
+  height: 91.5vh;
+  color: #FFF;
+  background-color: #46424D;
+}
+
+.h2 {
+  font-size: 1.5rem;
+  font-weight: 300;
+  color: #CBC9CF;
 }
 
 /* ---------------------NEW ACTIVITY SECTION--------------- */
 .new-activity {
-    /* border: 0.5em solid black; */
-    display: flex;
-    flex-direction: column;
-    width: 60%;
-    background-color:#2B2733;
+  /* border: 0.5em solid black; */
+  border-right: 0.25em solid #46424D;
+  width: 67%;
+  padding: 0 25px 25px 35px;
+  background-color:#2B2733;
 }
 
-.category-container {
-    /* border: 0.5em solid black; */
-    border-radius: 5px;
-    display: flex;
-    flex-direction: column;
-    justify-content: space-evenly;
-    align-items:center;
-    height: 75vh;
-    width: 95%;
-    margin-left: 1em;
-    background-color: #46424D;
+form {
+  /* border: 0.5em solid black; */
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  height: 75vh;
+  width: 100%;
+  background-color: #46424D;
 }
 
-h3 {
-    width: 47vw;
+.h3 {
+  font-size: 1.2rem;
+  font-weight: 400;
 }
 
-.categories {
-    /* border: 0.5em solid red; */
-    display: flex;
-    flex-direction: row;
-    justify-content: space-around;
-    width: 47vw;
+.category-box-wrapper {
+  display: flex;
+  justify-content: space-between;
+  width: 45vw;
 }
 
-.study-box {
-    /* border: 0.5em solid red;
-    color: red; */
-    margin-right: auto;
+.category-boxes {
+  border: 0.15em solid#FFF;
+  border-radius: 5px;
+  width: 10em;
+  text-align: center;
 }
 
-.exercise-box {
-    /* border: 0.5em solid red;
-    color: red; */
-    margin-left: auto;
+.category-image {
+  width: 1.5em;
+  margin-top: 10px;
 }
 
-.category-boxes {
-    border: 0.1em solid#FFF;
-    width: 10em;
-    text-align: center;
+.category-text {
+  font-size: 1.1rem;
+  font-weight: 300;
+  margin: 0 0 10px 0;
 }
 
-.category-image {
-    width: 2em;
-    text-align: center;
+.accomplish {
+  /* border: 0.5em solid blue; */
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  width: 45vw;
 }
 
-.accomplish-input {
-    /* border: 0.5em solid blue; */
-    display: flex;
-    flex-direction: column;
-    justify-content: space-evenly; 
-    width: 47vw;
-    /* margin-left: 2.8em;    */
+.time {
+  /* border: 0.5em solid green; */
+  display: flex;
+  width: 45vw;
+  justify-content: space-between;
+}
+
+.time-container {
+  flex-basis: 40%;
 }
 
-.time-input {
-    /* border: 0.5em solid green; */
-    display: flex;
-    width: 47vw;
-    justify-content:space-between;
+.minutes-input,
+.seconds-input,
+.accomplish-input {
+  border-top: none;
```

Comment:
Can we consolidate all border top et al into 1 line?

Reply:
yes! done